src/types/AceValidations.ts

```diff
@@ -116,9 +116,10 @@ const parseCheck = (yaml: string, check: ObjectCheck): AceCheck => {
     let path = check.path;
     if (path.startsWith('spec/destination')) {
       aceMarker = parseMarker(yaml, 'destination:');
-    } else if (path.startsWith('spec/name')) {
-      // This usecase comes from Destination Rule, the spec/name in the original yaml object is brought into kiali as destinationName field
-      aceMarker = parseMarker(yaml, 'destinationName:');
+    } else if (path.startsWith('spec/hosts')) {
```

Comment:
this is just a suggestion and not blocking the PR, but perhaps it would be worth to generalize this part of code? For instance, change parseMarker to receive generically a part of string without hardcoding it? At the moment there is a lot of very similar code (and might get more similar conditions in the future)

Reply:
This function should calculate the position of the line from a backend description.
There are some cases that could be unified, but I decided to not do it for readability as there are others that need special handling.